layers/+tools/google-calendar/packages.el

```diff
@@ -0,0 +1,89 @@
+;;; packages.el --- google-calendar layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Markus Johansson <markus.johansson@me.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+(defconst google-calendar-packages
+  '(org-gcal
+    calfw))
+
+(defun google-calendar/init-org-gcal ()
+  "Initializes org-gcal and adds keybindings for it's exposed functions"
+  (use-package org-gcal
+    :init
+    (progn
+      (spacemacs/set-leader-keys
+        "agr" 'org-gcal-refresh-token
+        "ags" 'org-gcal-sync
+        "agf" 'org-gcal-fetch))
+    :config
+    (setq org-gcal-down-days 365)   ;; Set org-gcal to download events a year in advance
```

Comment:
I probably wouldn't want this as the default. Maybe this can just be documented as something that can change.